src/generators/typescript/TypeScriptRenderer.ts

```diff
@@ -77,6 +77,11 @@ ${lines.map(line => ` * ${line}`).join('\n')}
       const rendererProperty = await this.runPropertyPreset(propertyName, property, this.model);
       content.push(rendererProperty);
     }
+    
+    if (this.model.additionalProperties !== undefined && this.model.additionalProperties instanceof CommonModel) {
+      const additionalProperty = await this.runAdditionalPropertiesPreset(this.model.additionalProperties, this.model);
```

Comment:
`additionalProperties` preset should also render another properties like patternProperties - also passed by options to preset - so this part should look like:

```ts
// from this
if (this.model.additionalProperties !== undefined && this.model.additionalProperties instanceof CommonModel) {
  const additionalProperty = await this.runAdditionalPropertiesPreset(this.model.additionalProperties, this.model);
  content.push(additionalProperty);
}

// to 
const additionalProperties = await this.runAdditionalPropertiesPreset(this.model.additionalProperties, this.model);
if (additionalProperties) {
  content.push(additionalProperties);
}
```

we shouldn't hardcoded only for `additionalProperties` from model.